backend/hitas/tests/apis/helpers.py

```diff
@@ -57,7 +57,12 @@ def path_pattern(self) -> Optional[str]:
 
 
 class HitasAPIClient(APIClient):
-    def generic(self, method, path, data="", content_type="json", secure=False, **kwargs) -> Response:
+    def generic(
+        self, method, path, data="", content_type="json", secure=False, openapi_validate=True, **kwargs
+    ) -> Response:
         response: Response = super().generic(method, path, data, content_type, secure=False, **kwargs)
-        validate_openapi(response)
+
+        if openapi_validate:
+            validate_openapi(response)
```

Comment:
Why would validating the api not be wanted? 🤔

Reply:
this is currently used when testing an endpoint that does not exist (then openapi validation would fail as well)